docs/content/environment_tests.md

```diff
@@ -103,9 +105,10 @@ performance_benchmark(env)
 
 The save observation test is to visually inspect the observations of games with graphical observations to make sure they are what is intended. We have found that observations are a huge source of bugs in environments, so it is good to manually check them when possible. This test just tries to save the observations of all the agents. If it fails, then it just prints a warning. The output needs to be visually inspected for correctness.
 
-``` python
+``` python notest
 from pettingzoo.test import test_save_obs
 from pettingzoo.butterfly import pistonball_v6
 env = pistonball_v6.env()
+#TODO running test_save_obs creates multiple files each time doctests are run here
```

Comment:
Wym by creates multiple files? Is it strictly a problem? If it’s possible to test that the commands work I’d prefer that even if it’s not ideal to create files

Reply:
I'll just allow this as we discussed on discord